web/wwwroot/js/site.js

```diff
@@ -1,5 +1,25 @@
 ﻿let currentTab = 0;
-showTab(currentTab);
+
+// Get the input field
+let inputs, currentInput
+if (window.location.pathname === "/landlord/register" || window.location.pathname === "/landlord/edit") {

```

Comment:
I'm wondering if it might be nicer to have this as a function which is called from within a `<script>` tag on each of these pages?

Reply:
I did try this at one point a few versions ago and it did seem to break it, however it might work now that I've refactored some of the code. I can give it another go :)

Reply:
Yeah unfortunately if i ever try and call a function from the script tag in one of these pages it always tells me the function is not defined...